kubernetes/cray-postgresql/templates/postgresql.yaml

```diff
@@ -0,0 +1,96 @@
+{{/*
+MIT License
+
+(C) Copyright [2022-2023] Hewlett Packard Enterprise Development LP
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+*/}}
+{{- if .Values.sqlCluster.enabled -}}
+{{- $root := . -}}
+---
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+metadata:
+  name: "{{ include "cray-postgresql.fullname" . }}-postgres"
+  labels:
+    app.kubernetes.io/name: {{ include "cray-postgresql.fullname" . }}-postgres
+    {{- include "cray-postgresql.common-labels" . | nindent 4 }}
+    {{- with .Values.sqlCluster.labels -}}
+    {{ toYaml . | nindent 4 -}}
+    {{- end }}
+spec:
```

Comment:
And for installs/upgrades - we'll want to add support enableLogicalBackup & logicalBackupSchedule in the postgresql.yaml and have this be something that the consuming chart can set as needed - again this can be a different ticket.

Reply:
Added support for enableLogicalBackup & logicalBackupSchedule